recipes/mender-shell/debian-master/control

```diff
@@ -0,0 +1,12 @@
+Source: mender-shell
+Maintainer: Mender Team <mender@northern.tech>
+Section: misc
```

Comment:
@kacf Thanks for bringing this up. I used `misc` same as we currently have for `mender-client` in this repo.

However now I have checked with Debian official packages, and `mender-artifact`, `mender-cli` and `mender-client` use `devel` (ref [here](https://salsa.debian.org/go-team/packages/mender-client/-/blob/master/debian/control#L2)).

I suggest then we change our packages to use `devel` too. Do you agree?

Reply:
I changed both `mender-client` (masetr) and `mender-shell` to `devel`.

Reply:
`mender-artifact` and `mender-cli` are clearly `devel`, since they are used to build and administer software for distribution. But I'm less sure about `mender-client` and `mender-shell`. Perhaps they fit better under the `utils` section? See [this list](https://packages.debian.org/stable/) for an explanation of the sections. They use the long name in the headers, but you can see the short name by hovering over the link.

Reply:
@kacf `utils` seems better than `devel`. I also checked RAUC package (see `apt-cache show rauc`), and they have their updater under `admin` section. Maybe admin is the best?

Reply:
Yes, that's a good category.